client/consul_test.go

```diff
@@ -0,0 +1,57 @@
+package client
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestSelf_SKU(t *testing.T) {
+	t.Parallel()
+
```

Comment:
Would this be better as a table?

Reply:
yeah, I just copied the test from [Nomad ](https://github.com/hashicorp/nomad/blob/d2b69b61da452ef71a14b245134186a4c3b48a75/command/agent/consul/self_test.go#L17) so didn't bother touching it. I can refactor it into a table.